evap/staff/importers.py

```diff
@@ -417,6 +464,83 @@ def process(cls, excel_content, test_run):
                 raise
 
 
+class PersonImporter:
+    def __init__(self):
+        self.success_messages = []
+        self.warnings = defaultdict(list)
+        self.errors = []
+
+    def process_participants(self, course, test_run, file_content=None, source_course=None):
+        if file_content:
+            user_list, self.success_messages, self.warnings, self.errors = UserImporter.process(file_content, test_run)
+        elif source_course:
+            user_list = list(source_course.participants.all())
+
+        usernames = [user.username for user in user_list]
+        already_related = course.participants.filter(username__in=usernames).all()
```

Comment:
i noticed that each element of user_list is either in already_related or users_to_add, right? i thought it might be nice to make that explicit. [relevant SO post](http://stackoverflow.com/questions/949098/python-split-a-list-based-on-a-condition). i thought about this :
```
already_related = []
users_to_add = []
participants = course.participants
already_related.append(user) for user in user_list if user in participants else users_to_add.append(user)
```
but it's a bit weird.

alternatively:
```
already_related = [user for user in user_list if user in course_participants]
users_to_add = [user for user in user_list if user not in course_participants]
```
takes double the time but i don't mind actually. what do you think?